src/content/docs/using-new-relic/welcome-new-relic/get-started/glossary.mdx

```diff
@@ -599,14 +599,14 @@ The length of time in seconds to wait for the aggregation window to fill with da
   >
   Insights was the name for the New Relic product that previously governed the reporting of custom events, as well as the ability to query and chart your New Relic data. These features are now a fundamental part of the New Relic One platform and are no longer governed by the Insights product or name. To learn more about these features:
 
-  * [Event API](/docs/telemetry-data-platform/ingest-apis/introduction-event-api) for reporting custom events
-  * [Query and chart data](/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder)
+    * [Event API](/docs/telemetry-data-platform/ingest-apis/introduction-event-api) for reporting custom events
+    * [Query and chart data](/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder)
 
   For historical reasons, the word "Insights" is still used in some places. For example:
```

Comment:
In my local build this line appears indented. Minor thing.

Reply:
I couldn't figure out the indents here either! Very frustrating. I'll merge for now but might poke at iit more later.